content/en/docs/contribute/generate-ref-docs/contribute-upstream.md

```diff
@@ -240,7 +240,6 @@ You are now ready to follow the [Generating Reference Documentation for the Kube
 * [Generating Reference Documentation for the Kubernetes API](/docs/contribute/generate-ref-docs/kubernetes-api/)
 * [Generating Reference Docs for Kubernetes Components and Tools](/docs/home/contribute/generated-reference/kubernetes-components/)
 * [Generating Reference Documentation for kubectl Commands](/docs/home/contribute/generated-reference/kubectl/)
-* [Generating Reference Documentation for the Kubernetes Federation API](/docs/home/contribute/generated-reference/federation-api/)
```

Comment:
Thanks! Blast it, another small nit, line 233, extra `to`:
`You are now ready to follow the [Generating Reference Documentation for the Kubernetes API](/docs/contribute/generate-ref-docs/kubernetes-api/) guide to to generate the`
/lgtm